### PR TITLE
Reading datasources from ftp too

### PR DESCRIPTION
**Pull request by dmvieira**

### CODE REVIEW

**Comment by dmvieira** [comment]
This PR closes #5

File: dataset_manager/data_source.py [hunk 1]
```diff
@@ -5,7 +5,11 @@
 module to prepare the datasets
 """
 import os
-import requests
+try:
+    import urllib2
+except Exception:
```
Comment (sbouchardet) [inline dataset_manager/data_source.py]: Is there a more specific Exception to catch here?

**Review by sbouchardet — COMMENTED** [review]
What version of python broke the import? Was the try/catch the only option?

File: dataset_manager/data_source.py (hunk 1 above)
Comment (dmvieira) [reply dataset_manager/data_source.py]: Python 3.3-4 and >= 3.5 raise different exception for import error

Comment (dmvieira) [reply dataset_manager/data_source.py]: I think there's another way to validate python 2 or 3 for import... I'll check

**Review by sbouchardet — APPROVED** [review]
LGTM

**Pull request merged**